.github/tekton/pull-request.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: ks-pull-request-pipeline
+spec:
+  workspaces:
+    - name: repo
+  params:
+    - name: repo-full-name
+      description: "Repository full name. like: kubesphere-sigs/ks"
+    - name: clone-url
+      description: Git repository clone URL.
+    - name: revision
+      description: Git repository revision to checkout.
+    - name: dashboard-url
+      description: Tekton dashboard access URL.
+      default: "http://demo:31962/#/namespaces/ks/pipelineruns"
+  tasks:
+    - name: set-running-status
+      taskRef:
+        name: github-set-status
+      params:
+        - name: REPO_FULL_NAME
+          value: $(params.repo-full-name)
+        - name: SHA
+          value: $(params.revision)
+        - name: DESCRIPTION
+          value: Build has started
+        - name: STATE
+          value: pending
+        - name: TARGET_URL
+          value: $(params.dashboard-url)
```

Comment:
Is that possible to make it be a specific URL instead of the dashboard?

Reply:
At present, it's hard to obtain a specific URL here. I'd prefer to figure it out in the future.